core-galleon-pack/src/main/resources/feature_groups/domain-profile.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<feature-group-spec name="domain-profile" xmlns="urn:jboss:galleon:feature-group:1.0">
```

Comment:
I think the legacy domain profile is wrong. It's missing the discovery and security-manager subsystems and I don't see any good reason for that.

That probably doesn't change the structure of your feature groups much though.  Your basic-profile seems like a reasonable base, with discovery and security-manager kind of extra. And then elytron seems to have different detail settings between standalone and domain.

Reply:
https://issues.jboss.org/browse/WFCORE-3784

I'll send up a PR fixing that for legacy and then paste links here to a branch that incorporates that plus a tweak to this file to make it align.

Reply:
The last two commits on https://github.com/wildfly/wildfly-core/compare/master...bstansberry:WFCORE-3769_3784 correct that bug and update the galleon domain-profile.xml to match.

#3240 is an independent PR that just fixes legacy.

Reply:
The fix for domain-profile.xml has been cherry-picked into this branch. Thanks!